tests/search/search_export_tests.py

```diff
@@ -36,39 +46,54 @@ def setUpTestData(cls):
         ResourceGraphImporter(archesfile["graph"])
 
         cls.search_model_graphid = "d291a445-fa5f-11e6-afa8-14109fd34195"
-        # cls.search_model_cultural_period_nodeid = "7a182580-fa60-11e6-96d1-14109fd34195"
-        # cls.search_model_creation_date_nodeid = "1c1d05f5-fa60-11e6-887f-14109fd34195"
-        # cls.search_model_destruction_date_nodeid = "e771b8a1-65fe-11e7-9163-14109fd34195"
+        cls.search_model_cultural_period_nodeid = "7a182580-fa60-11e6-96d1-14109fd34195"
+        cls.search_model_cultural_period_nodename = "Cultural Period Concept"
         cls.search_model_name_nodeid = "2fe14de3-fa61-11e6-897b-14109fd34195"
-        # cls.search_model_sensitive_info_nodeid = "57446fae-65ff-11e7-b63a-14109fd34195"
-        # cls.search_model_geom_nodeid = "3ebc6785-fa61-11e6-8c85-14109fd34195"
 
         cls.user = User.objects.create_user(
             "unprivileged_user", "unprivileged_user@test.com", "test"
         )
 
-        test_resourceinstanceid = uuid.uuid4()
+        cls.test_resourceinstanceid = uuid.uuid4()
 
         cls.loadOntology()
         cls.ensure_resource_test_model_loaded()
         models.ResourceInstance.objects.get_or_create(
             graph_id=cls.search_model_graphid,
-            resourceinstanceid=test_resourceinstanceid,
+            resourceinstanceid=cls.test_resourceinstanceid,
         )
         tile_data = {}
         tile_data[cls.search_model_name_nodeid] = {
             "en": {"value": "Etiwanda Avenue Street Trees", "direction": "ltr"}
         }
         new_tile = Tile(
-            resourceinstance_id=test_resourceinstanceid,
+            resourceinstance_id=cls.test_resourceinstanceid,
             data=tile_data,
             nodegroup_id=cls.search_model_name_nodeid,
         )
         new_tile.save()
-        time.sleep(1)  # delay to allow for async indexing
-
+        skos = SKOSReader()
+        rdf = skos.read_file("tests/fixtures/data/concept_label_test_scheme.xml")
+        ret = skos.save_concepts_from_skos(rdf)
+
+        skos = SKOSReader()
+        rdf = skos.read_file("tests/fixtures/data/concept_label_test_collection.xml")
+        ret = skos.save_concepts_from_skos(rdf)
+        cls.valueid = "dadaeee5-57ef-409d-a6cf-98d332fdada8"
+        cultural_period_tile = Tile(
+            data={cls.search_model_cultural_period_nodeid: cls.valueid},
+            nodegroup_id=cls.search_model_cultural_period_nodeid,
+            resourceinstance_id=cls.test_resourceinstanceid,
+        )
+        cultural_period_tile.save(index=False)
+        cultural_period_tile.index()
+        sync_es(se)
```

Comment:
Nice. I'm wondering if this belongs in setUp() so it runs before each test instead of once per class.

Reply:
do you mean overriding setUp in the SearchExportTests class or in the base class method?

Reply:
In SearchExportTests

Reply:
Sorry Galen, I meant the regular instance method, not a classmethod. I'm not sure a classmethod like this is ever called?

Reply:
I just pushed the change you mentioned. I'm fairly certain the classmethod was being called as tests pass with it as a classmethod or instancemethod but fail without either.